tests/app/test_utilities.py

```diff
@@ -1,248 +1 @@
-        self.assertEqual(cc, 'CollectionCase')
-        self.assertEqual(label, 'INBOX')
-        self.assertEqual(desc, False)
-        self.assertEqual(ce, 'CollectionExercise')
-
-    def test_add_string_query_no_args(self):
-        """Adding args to empty string query"""
-        string_query_args = '?'
-        string_query_args = utilities.add_string_query_args(string_query_args, 'ru_id', '7fc0e8ab-189c-4794-b8f4-9f05a1db185b')
-        self.assertEqual(string_query_args, '?ru_id=7fc0e8ab-189c-4794-b8f4-9f05a1db185b')
-
-    def test_add_string_query_with_args(self):
-        """Adding args to string query with arg"""
-        string_query_args = '?survey=BRES'
-        string_query_args = utilities.add_string_query_args(string_query_args, 'ru_id', '7fc0e8ab-189c-4794-b8f4-9f05a1db185b')
-        self.assertEqual(string_query_args, '?survey=BRES&ru_id=7fc0e8ab-189c-4794-b8f4-9f05a1db185b')
-
-    def test_generate_etag_with_none_in_msg_to(self):
-        """Generating etag when msg_to has a None value"""
-
-        generated_etag = generate_etag([], 'd740-10d5-4ecb-b', 'subject', 'body')
-
-        msg_to = []
-        msg_to_str = ' '.join(str(uuid) for uuid in msg_to)
-
-        data_to_hash = {
-            'msg_to': msg_to_str,
-            'msg_id': 'd740-10d5-4ecb-b',
-            'subject': 'subject',
-            'body': 'body'
-        }
-
-        hash_object = hashlib.sha1(str(sorted(data_to_hash.items())).encode())
-        etag = hash_object.hexdigest()
-
-        self.assertEqual(generated_etag, etag)
-
-    def test_generate_etag_with_msg_to(self):
-        """Generating etag when msg_to has a uuid value"""
-
-        generated_etag = generate_etag(['7fc0e8ab-189c-4794-b8f4-9f05a1db185b'], 'd740-10d5-4ecb-b', 'subject', 'body')
-
-        msg_to = ['7fc0e8ab-189c-4794-b8f4-9f05a1db185b']
-        msg_to_str = ' '.join(str(uuid) for uuid in msg_to)
-
-        data_to_hash = {
-            'msg_to': msg_to_str,
-            'msg_id': 'd740-10d5-4ecb-b',
-            'subject': 'subject',
-            'body': 'body'
-        }
-
-        hash_object = hashlib.sha1(str(sorted(data_to_hash.items())).encode())
-        etag = hash_object.hexdigest()
-
-        self.assertEqual(generated_etag, etag)
-
-    def test_generate_etag_with_multiple_msg_to(self):
-        """Generating etag when msg_to has a uuid value"""
-
-        generated_etag = generate_etag(['1', '2', '3'], 'd740-10d5-4ecb-b', 'subject', 'body')
-
-        msg_to = ['1', '2', '3']
-        msg_to_str = ' '.join(str(uuid) for uuid in msg_to)
-
-        data_to_hash = {
-            'msg_to': msg_to_str,
-            'msg_id': 'd740-10d5-4ecb-b',
-            'subject': 'subject',
-            'body': 'body'
-        }
-
-        hash_object = hashlib.sha1(str(sorted(data_to_hash.items())).encode())
-        etag = hash_object.hexdigest()
-
-        self.assertEqual(generated_etag, etag)
+BRES_SURVEY ="33333333-22222-3333-4444-88dc018a1a4c"
```

Comment:
This doesn't seem like the right place to keep this.

The location/name of the file implies it should contain tests